scripts/freefood.coffee

```diff
@@ -0,0 +1,52 @@
+# Description:
+#   Announce free food.
+#
+# Commands:
+#   <botname> free <food> <location>
+
+module.exports = (robot) ->
+  robot.respond /free (.+)/i, (msg) ->
```

Comment:
Yeah agreed, I almost have it working and that's what I ended up deciding as well :P